engines/bops_api/spec/requests/v2/public/documents_spec.rb

```diff
@@ -42,4 +42,58 @@
       end
     end
   end
+
+  context "when there is a site_notice" do
+    let!(:site_notice) do
+      create(:site_notice,
+        planning_application: planning_application,
+        required: true,
+        displayed_at: "2024-01-08T09:00:00Z",
+        expiry_date: "2024-01-30",
+        internal_team_email: "pressteam@example.com")
+    end
+
+    let!(:site_notice_evidence) do
+      create(:document,
+        planning_application: planning_application,
+        owner: site_notice,
+        file: fixture_file_upload("site-notice.jpg", "image/jpeg", true),
+        tags: ["internal.siteNotice"])
+    end
+
+    path "/api/v2/public/planning_applications/{reference}/documents" do
+      get "Retrieves documents for a planning application" do
+        tags "Planning applications"
+        produces "application/json"
+
+        parameter name: :reference, in: :path, schema: {
+          type: :string,
+          description: "The planning application reference"
+        }
+
+        response "200", "returns a planning application's documents and decision notice given a reference" do
+          example "application/json", :default, example_fixture("documents.json")
+          schema "$ref" => "#/components/schemas/Documents"
+
+          let(:reference) { planning_application.reference }
+          let(:planning_application) { create(:planning_application, :published, :with_boundary_geojson, :determined, documents: [document], local_authority:, application_type:) }
+
+          run_test! do |response|
+            data = JSON.parse(response.body)
+            expect(data["application"]["reference"]).to eq(planning_application.reference)
+            expect(data["files"]).not_to be_empty
```

Comment:
Should this check whether the site notice is included in `data["files"]`?

Otherwise LGTM